packages/framework/src/servers/lambda.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  type APIGatewayEvent,
+  type APIGatewayProxyEventV2,
+  type APIGatewayProxyResult,
+  type Context,
+} from 'aws-lambda';
+import { NovuRequestHandler, type ServeHandlerOptions } from '../handler';
+import { type SupportedFrameworkName, type Either } from '../types';
+
+export const frameworkName: SupportedFrameworkName = 'lambda';
+
+export const serve = (options: ServeHandlerOptions) => {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Add the missing return typing‏
```suggestion
export const serve = (options: ServeHandlerOptions): Promise<APIGatewayProxyResult>) => {
```

Reply:
this was causing very strange type errors, so I decided to leave it out for the time being :(